autonomy/testing.py

```
@@ -0,0 +1,1027 @@
+            await asyncio.sleep((worker_id / concurrent_users) * ramp_up_seconds)
+
+            while time.time() - start_time < duration_seconds:
+                call_start = time.time()
+                try:
+                    if asyncio.iscoroutinefunction(target_function):
+                        await target_function()
+                    else:
+                        target_function()
+                    duration = (time.time() - call_start) * 1000
+                    worker_results.append({"duration_ms": duration, "success": True})
+                except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)